jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/VolumePermissionEnricher.java

```diff
@@ -178,6 +187,33 @@ private String getMountPath(List<Container> containers, String name){
                 throw new IllegalArgumentException("No matching volume mount found for volume "+ name);
             }
 
+            private ResourceRequirements createNewResourceRequirements() {
+                if (hasNoResourcesConfigured())
+                    return null;
```

Comment:
Please, use curly braces with if-else statements.
This should be part of our style guidelines (personally it's the only thing I need enforced since I've faced very long debugging sessions caused by an if-else statement lacking curly braces and with wrong indentation.